app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -149,6 +204,63 @@ class PoliciesActivityTest {
       }
     }
   }
+  @Test
+  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
```

Comment:
What's actually being tested here? The test name is missing the action being done.

Reply:
I have removed this test.